db/schema.rb

```diff
@@ -10,74 +10,8 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_09_19_064605) do
+ActiveRecord::Schema[7.0].define(version: 0) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "active_storage_attachments", force: :cascade do |t|
-    t.string "name", null: false
-    t.string "record_type", null: false
-    t.bigint "record_id", null: false
-    t.bigint "blob_id", null: false
-    t.datetime "created_at", null: false
-    t.index ["blob_id"], name: "index_active_storage_attachments_on_blob_id"
-    t.index ["record_type", "record_id", "name", "blob_id"], name: "index_active_storage_attachments_uniqueness", unique: true
-  end
-
-  create_table "active_storage_blobs", force: :cascade do |t|
-    t.string "key", null: false
-    t.string "filename", null: false
-    t.string "content_type"
-    t.text "metadata"
-    t.string "service_name", null: false
-    t.bigint "byte_size", null: false
-    t.string "checksum"
-    t.datetime "created_at", null: false
-    t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
-  end
-
-  create_table "active_storage_variant_records", force: :cascade do |t|
-    t.bigint "blob_id", null: false
-    t.string "variation_digest", null: false
-    t.index ["blob_id", "variation_digest"], name: "index_active_storage_variant_records_uniqueness", unique: true
-  end
-
-  create_table "journal_entries", force: :cascade do |t|
-    t.integer "journal_id", null: false
-    t.string "title", null: false
-    t.text "content", null: false
-    t.text "goal"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.index ["journal_id"], name: "index_journal_entries_on_journal_id"
-  end
-
-  create_table "journals", force: :cascade do |t|
-    t.string "title", null: false
-    t.text "description", null: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.integer "user_id", null: false
-  end
-
-  create_table "users", force: :cascade do |t|
-    t.string "username", null: false
-    t.string "first_name"
-    t.string "last_name"
-    t.string "password_digest", null: false
-    t.string "password_confirmation"
-    t.text "description"
-    t.string "email"
-    t.integer "mobile"
-    t.string "adress"
-    t.string "country"
-    t.string "city"
-    t.date "date_of_birth"
-    t.date "joining_date"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-  end
-
-  add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
-  add_foreign_key "active_storage_variant_records", "active_storage_blobs", column: "blob_id"
 end
```

Comment:
what happened here ?

Reply:
AS I'm working on Branches with different migrations, I normally migrate down in one branch to successfully migrate up in the other one. If i dont do so i get errors migrating them, because rows and tables are missing